common/DevHomeAdaptiveCards/CardInterfaces/IDevHomeSettingsCardNonSubmitAction.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using AdaptiveCards.ObjectModel.WinUI3;
+using CommunityToolkit.Mvvm.Input;
+
+namespace DevHome.Common.DevHomeAdaptiveCards.CardInterfaces;
+
+/// <summary>
+/// Represents an action that can be invoked from a Dev Home settings card that does not submit the card. E.g launches a dialog.
+/// </summary>
+public interface IDevHomeSettingsCardNonSubmitAction : IAdaptiveCardElement
```

Comment:
What about instead of a submit action and a non-submit action, there's one action where the Verb is either Submit or something else?

Reply:
I chose non-submit here because, I wanted to be explicit about the action since almost all the buttons that get rendered in an adaptive card are backed by an action interface that submits the card. In my case I have button but it doesn't submit the adaptive card. It opens a content dialog but I'll note this down and we can refine the naming later.